### PR TITLE
Feature/connected demo

### PR DESCRIPTION
**Pull request by JohnLynnBeck**
Adding documentation to the docs app for JSS Connected Demo.

### CODE REVIEW

File: docs/package.json [hunk 1]
```diff
@@ -68,6 +68,7 @@
     "react-helmet": "^5.2.0",
     "react-router": "^4.3.0",
     "react-router-dom": "^4.3.1",
+    "react-router-hash-link": "^1.2.1",
```
Comment (anastasiya29) [inline docs/package.json]: Do we still need to add this new module? I believe we ended up removing all the hash links.

File: docs/assets/css/sidenav.css [hunk 2]
```diff
@@ -45,4 +45,73 @@
     height: 25px;
     color: black;
     font-weight: bold;
+}
+
```
Comment (anastasiya29) [inline docs/assets/css/sidenav.css]: Originally the new sidenav had different styling from the docs sidenav, but the end result is that both side navs use the original styling. Can you confirm that this new css is still needed? Or is this leftover and needs to be removed?

File: docs/package.json (hunk 1 above)
Comment (JohnLynnBeck) [reply docs/package.json]: Ah, after the sidenav reorg, we do not. I'll remove.

File: docs/assets/css/sidenav.css (hunk 2 above)
Comment (JohnLynnBeck) [reply docs/assets/css/sidenav.css]: Definitely not needed. Thought I'd deleted it already, as a matter of fact.

**Comment by aweber1** [comment]
I'm going to be super annoying, but can you close this PR and branch off of `master` instead of `dev` for these changes? Then open a PR to merge into `master`.

I think we need to change the process for making non-feature-related docs changes.

**Comment by aweber1** [comment]
Closing in favor of #180

**Pull request closed without merging**